gui/src/app/compileStanProgram/compileStanProgram.ts

```diff
@@ -45,7 +45,8 @@ const compileStanProgram = async (stanWasmServerUrl: string, stanProgram: string
             }
         });
         if (!c.ok) {
-            onStatus(`failed to run job: ${c.statusText}`);
+            let j = await c.json();
+            onStatus(`failed to run job: ${j.message ?? c.statusText}`);
```

Comment:
Sorry, probably best to do `j?.message ?? c.statusText` where the `?.` is a null propagation operator (not sure if that's the official name)--otherwise there's a potential error in the case where `j` is undefined, and thus has no members, so `j.message` cannot resolve.